docs/sidebars.js

```diff
@@ -46,13 +45,15 @@ module.exports = {
         "migration/global-variables",
         // "migration/experimental-components",
         // "migration/state-management",
+        "guides/rag-with-astradb",
```

Comment:
I'm not sure about this. The guides for the starter projects should be the first step after installing Langflow, IMO. What do you think?

Reply:
Agreed, added to sidebar